code/game/turfs/flooring/flooring_vr.dm

```diff
@@ -26,4 +26,37 @@
 	icon_base = "grass-dark"
 
 /turf/simulated/floor/tiled/freezer/cold
-	temperature = T0C - 5
\ No newline at end of file
+	temperature = T0C - 5
+
+/turf/simulated/floor/redgrid
+	name = "processing strata"
+	icon = 'icons/turf/flooring/circuit_vr.dmi'
+	icon_state = "rcircuit"
+	initial_flooring = /decl/flooring/reinforced/circuit/red
```

Comment:
Did we not already have the red grid circuit floors? I thought we did

Reply:
Nope we did not!